readthedocs/projects/views/private.py

```diff
@@ -553,12 +557,15 @@ def post(self, request, *args, **kwargs):
             self.get_queryset(),
             username=username,
         )
-        if user == request.user:
+        if self._is_last_user():
             raise Http404
```

Comment:
yeah, but this is outisede a form, not sure if we can display an error to the view level instead of the form level. Also this case also only happens if the user triggers the request manually or if another user has removed itself in another request, the UI doesn't show you the option if you are the only one.

Let me know if we should still display an error, maybe in the 404?

Reply:
Wouldn't this be a 400 saying that you can't remove yourself if you are the last/only user of the project?

Reply:
Could this be validated in the form?

Reply:
We don't have a form here, this is just a normal view (we don't display the option to remove yourself if you are the only one left). I have changed the status code to 400.